### ISSUE TITLE
JSON Scenario Template

### ISSUE THREAD

**Issue by simnh** [energiekollektiv/openmod.sh#14]
General open question for terminology: 

* energy_sector vs. sector, energy_amount vs. amount?
* Inlcude 'units' as children of scenario?
* One emissions element per emission as a child of scenario element?
* For flexible generator only include 'output'-hub in hubs and define input
  hub from fueltype. Or only from fueltype if only one, i.e. the output hub is
  given? 
* variable_cost vs. variable_costs vs. marginal_cost ? 
* include price similar to emissions and units? (global prices instead of 
variable costs, or variable costs except fuel costs?)
* oemof_class / pypsa_class (keep as tag? or one level above?, not needed for template...)

**Comment by cswh** [comment]
do we still have open questions here? or do we "only" need to document the decisions we have taken how we deal with the points?

**Comment by simnh** [comment]
I think we made most choices implicitly, generally we went from a little more concrete to a little more abstract (e.g. energy_sector -> sector). If thats fine for you? 
But documentation is really important. I already started that in the api documentation, where I wanted to document all different element - 'types' (e.g. flexible_transformer, commodity...etc.). And how the connection between them is arranged
